src/main.c

```diff
@@ -255,6 +258,17 @@ int main(int argc, char **argv)
 				break;
 			}
 			if (strcmp(long_options[option_index].name,
+			           "loop") == 0) {
+				long int loop = strtol(optarg, NULL, 0);
+				if (loop < 0) {
+					log_warn("Bad loop option: " \
+					         "\"%s\"\n", optarg);
+					break;
+				}
+				cfg.loop = loop;
```

Comment:
A `long` is cast into an `int`. I'm not programming much in C these days, but couldn't this be an issue if `loop` is insanely large?

Reply:
very large values don't make much sense here. The value is used as argument for `sleep(unsigned int)`, so even in this call, negative values are (probably) cast to large positive numbers. The worst that can happen is that _openfortivpn_ waits a very long time before trying to reconnect, which gives a user experience nearly as if loop was left with the default value of 0. 

I don't feel that a code change is needed here, but I'm open for suggestions. Maybe some comments of [this discussion](https://stackoverflow.com/questions/7021725/converting-string-to-integer-c) are helpful

Reply:
That's not an important issue. Let's reconsider only if it raises a Coverity issue.